config/locales/server.en.yml

```diff
@@ -274,6 +274,7 @@ en:
   not_accepting_pms: "Sorry, %{username} is not accepting messages at the moment."
   max_pm_recepients: "Sorry, you can send a message to maximum %{recipients_limit} recipients."
   pm_reached_recipients_limit: "Sorry, you can't have more than %{recipients_limit} recipients in a message."
+  remove_useless_quotes: "Automatically removed quote of whole previous post."
```

Comment:
I would not call it "useless", instead "remove_direct_reply_full_quotes` useless has a very bad taste.